patterns/footer-large-dark.php

```diff
@@ -70,15 +70,15 @@ class="wp-block-group alignfull has-background-color has-white-color has-black-b
 	<div class="wp-block-group alignfull has-background-color has-text-color" style="padding-top:0px;padding-right:0px;padding-bottom:0px;padding-left:0px">
 		<!-- wp:group {"style":{"spacing":{"blockGap":"8px"},"elements":{"link":{"color":{"text":"var:preset|color|background"}}}},"textColor":"background","layout":{"type":"flex","flexWrap":"nowrap"}} -->
 		<div class="wp-block-group has-background-color has-text-color has-link-color">
-			<!-- wp:paragraph {"style":{"typography":{"fontSize":"12px"}}} -->
-			<p style="font-size:12px">@ 2022</p>
+			<!-- wp:paragraph -->
+			<p>@ <?php echo esc_html( gmdate( 'Y' ) ); ?></p>
 			<!-- /wp:paragraph -->
-			<!-- wp:site-title {"style":{"typography":{"fontStyle":"normal","fontWeight":"400","fontSize":"12px"}}} /-->
+			<!-- wp:site-title /-->
 		</div>
 		<!-- /wp:group -->
 
-		<!-- wp:paragraph {"style":{"typography":{"fontSize":"14px"}}} -->
-		<p style="font-size:14px"><em>Built with <a href="https://woocommerce.com/">WooCommerce</a> </em></p>
+		<!-- wp:paragraph -->
+		<p><em><?php esc_html_e( 'Built with', 'woo-gutenberg-products-block' ); ?> <a href="https://woocommerce.com/">WooCommerce</a> </em></p>
```

Comment:
I think to ease localization it's better if "WooCommerce" is part of the string to translate. Something along these lines:

```suggestion
		<p><em>
		<?php
			echo wp_kses_post(
				sprintf(
					__( 'Built with %1$sWooCommerce%2$s', 'woo-gutenberg-products-block' ),
					'<a href="https://woocommerce.com/">',
					'</a>'
				)
			)
			?>
		</em></p>
```

Reply:
Good call! Included the brand name on [8bfe19d](https://github.com/woocommerce/woocommerce-blocks/pull/10323/commits/8bfe19d11f927d0585318f33ac3d4cb9a39f364a).

ps: I did a change to rely on `wp_kses` instead of `wp_keses_post` for better performance.